### PR TITLE
Implemented Cartpole PETS version

### PR DESCRIPTION
**Pull request by hxri**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

A different version of Cartpole env is implemented that uses PETS with reference to https://github.com/kchua/handful-of-trials/blob/master/dmbrl/env/cartpole.py

This PR is related to issue #117. 

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](../../CONTRIBUTING.md) document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: mbrl/env/reward_fns.py [hunk 1]
```diff
@@ -13,6 +13,12 @@ def cartpole(act: torch.Tensor, next_obs: torch.Tensor) -> torch.Tensor:
     return (~termination_fns.cartpole(act, next_obs)).float().view(-1, 1)
 
 
+def cartpole_pets(act: torch.Tensor, next_obs: torch.Tensor) -> torch.Tensor:
```
[the remark this answers is not in the document]
Comment (hxri) [reply mbrl/env/reward_fns.py:17]:
```
def cartpole_pets(act: torch.Tensor, next_obs: torch.Tensor) -> torch.Tensor:
    assert len(next_obs.shape) == len(act.shape) == 2

    obs_cost = -torch.exp(
        -torch.sum(torch.square(next_obs[:, 0]) - torch.tensor([0.0, 0.6])), axis=1
        ) / (0.6 ** 2)
    act_cost = 0.01 * torch.sum(torch.square(act), axis=1)

    return (obs_cost + act_cost).view(-1, 1)
```

@luisenp 

I have made the changes you have mentioned. But I am not so sure about matching the batch dimension. Can you please check the above code and see if it is correct?

Thanks!

[the remark this answers is not in the document]
Comment (luisenp) [reply mbrl/env/reward_fns.py:17]: Thanks a lot for the changes. I'm busy today with other projects, but I'm planning to look at this tomorrow or Wednesday. Thanks!

**Comment by hxri** [comment]
@luisenp Is there anything that I can do?

**Comment by luisenp** [comment]
Hi @hxri. I need to take a look at this more deeply, which I'll start doing today or tomorrow. One thing that could be helpful is if you can run some tests on your side to confirm that the reward function is consistent with the real reward function of the environment.

**Review by luisenp — APPROVED** [review]

**Comment by luisenp** [comment]
HI @hxri, this works now. Thanks for the contribution! Closes #117

**Comment by hxri** [comment]
@luisenp Thank you.
